src/protocols/MeterS0.cpp

```diff
@@ -322,7 +322,7 @@ ssize_t MeterS0::read(std::vector<Reading> &rds, size_t n) {
 
 	if (_send_zero || t_imp > 0) {
 		if (!_first_impulse) {
-			double value = 3600000 / ((t2-t1) * (_resolution * t_imp));
```

Comment:
For clarity you could move resolution to the end of the term but it amounts to the same. I guess only seen now as local httpd is rarely used.